modules/drivers/lidar/lidar_robosense/proto/BUILD

```diff
@@ -0,0 +1,40 @@
+load("//tools:cpplint.bzl", "cpplint")
+
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "suteng_proto",
+    deps = [
+        ":sensor_suteng_proto",
```

Comment:
Please rewrite the proto BUILD file using proto_build_generator.py when this PR was merged into master.